stable/awsebscsiprovisioner/templates/statefulset.yaml

```diff
@@ -74,9 +74,7 @@ spec:
             - --provisioner=ebs.csi.aws.com
             - --csi-address=$(ADDRESS)
             - --v=5
-            {{- if .Values.enableVolumeScheduling }}
```

Comment:
this had to be `.Values.provisioner.enableVolumeScheduling`... that was the problem

Reply:
Makes sense. So it isn't any weird templating behavior. Great, I'll fix the value instead of removing it.